start_admin_sshd.sh

```diff
@@ -65,31 +67,31 @@ if trusted_user_ca_keys=$(get_user_data_keys "trusted_user_ca_keys"); then
   ((++available_auth_methods))
 fi
 
-chown "${local_user}" -R "${user_ssh_dir}"
+chown "${LOCAL_USER}" -R "${USER_SSH_DIR}"
 
 # If there were no successful auth methods, then users cannot authenticate
 if [[ "${available_auth_methods}" -eq 0 ]]; then
-  user_data_condensed=$(tr -d '[:space:]' < "${user_data}")
+  user_data_condensed=$(tr -d '[:space:]' < "${USER_DATA}")
   log "Failed to configure ssh authentication with user-data: ${user_data_condensed}"
   exit 1
 fi
 
 # Generate the server keys
-mkdir -p "${ssh_host_key_dir}"
+mkdir -p "${SSH_HOST_KEY_DIR}"
 for key in rsa ecdsa ed25519; do
```

Comment:
Please rename this var while you're here. These aren't "keys" but rather a list of "key algorithms" to generate:

```suggestion
for keyalg in rsa ecdsa ed25519; do
```